src/consumer.py

```diff
@@ -0,0 +1,89 @@
+import asyncio
+import os
+import struct
+from datetime import datetime
+
+import typer
+from fastapi import FastAPI
+from prometheus_client import Counter, Histogram, make_asgi_app
+from streamstore import S2
+from streamstore.schemas import SequencedRecord
+from uvicorn import Config, Server
+
+S2_AUTH_TOKEN = os.getenv("S2_AUTH_TOKEN")
+
+METRICS_NAMESPACE = "stress"
+
+e2e_latency_histogram = Histogram(
```

Comment:
What bins does this have? We may want to tailor it a bit to have more precision in the expected range

Reply:
default seems to be `(.005, .01, .025, .05, .075, .1, .25, .5, .75, 1.0, 2.5, 5.0, 7.5, 10.0, INF)`. what should we specify instead?

Reply:
I'm thinking `.010, .020, .030, .040, .050, .075, .100, .200, .300, .400, .500, .750, 1.0, 2.5, 5.0, INF`